Assets/MixedRealityToolkit/Utilities/CameraFOVChecker.cs

```diff
@@ -22,22 +22,23 @@ public static class CameraFOVChecker
         // List of corners shared across all sphere pointer query instances --
         // used to store list of corners for a bounds. Shared and static
         // to avoid allocating memory each frame
-        private static List<Vector3> inFOVConeBoundsCornerPoints = new List<Vector3>();
+        private static List<Vector3> inFOVBoundsCornerPoints = new List<Vector3>();
 
         /// <summary>
         /// Returns true if a collider's bounds is within the camera FOV. 
         /// Utilizes a cache to test if this collider has been seen before and returns current frame's calculated result.
+        /// NOTE: This is a 'loose' FOV check -- it can return true in cases when the collider is actually not in the FOV
+        /// because it does an axis-aligned check when testing for large colliders. So, if the axis aligned bounds are in the bounds of the camera, it will return true.
         /// </summary>
         /// <param name="myCollider">The collider to test</param>
-        public static bool IsInFOVConeCached(this Camera cam,
-            Collider myCollider)
+        public static bool IsInFOVCached(this Camera cam, Collider myCollider)
         {
             // if the collider's size is zero, it is not visible. Return false.
-            if(myCollider.bounds.size == Vector3.zero || myCollider.transform.localScale == Vector3.zero)
+            if (myCollider.bounds.size == Vector3.zero || myCollider.transform.localScale == Vector3.zero)
             {
                 return false;
             }
-            
+
             Tuple<Collider, Camera> cameraColliderPair = new Tuple<Collider, Camera>(myCollider, cam);
             bool result = false;
             if (inFOVConeLastCalculatedFrame != Time.frameCount)
```

Comment:
Nit: There are still some members named with "cone" - it might be worth clearing that name as well now that it's a non-cone bounds check (i.e. IsInFOVCached instead of Cone cached)

Reply:
Fixed.